paper-input-container.html

```diff
@@ -365,10 +366,14 @@
       var value = inputElement[this._propertyForValue] || inputElement.value;
 
       var valid;
-      if (inputElement.validate) {
-        valid = inputElement.validate(value);
-      } else if (inputElement.checkValidity) {
-        valid = inputElement.checkValidity();
+
+      // type="number" doesn't have a validate method...
+      if (this.autoValidate) {
+        if (inputElement.validate) {
+          valid = inputElement.validate(value);
+        } else {
+          valid = inputElement.checkValidity();
+        }
       } else {
         valid = true;
```

Comment:
Does that work with `<input type="number">`? The check in line 382 needs the actual validity. Try typing in some alphabetic characters input a number input.